db/migrate/20220916093407_create_order.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOrder < ActiveRecord::Migration[6.1]
+  def change
+    create_table :orders do |t|
+      t.integer :customer_id
+      t.decimle :total_price
+      t.integer :discount
+      t.decimle :discounted_value
+      t.decimle :after_discount
```

Comment:
after_discount is ok for our case, but we can also use net_amount

Reply:
changed into total_discount